app/models/preserved_copy.rb

```diff
@@ -8,23 +8,30 @@ class PreservedCopy < ApplicationRecord
   UNEXPECTED_VERSION_ON_STORAGE_STATUS = 'unexpected_version_on_storage'.freeze
   VALIDITY_UNKNOWN_STATUS = 'validity_unknown'.freeze
   UNREPLICATED_STATUS = 'unreplicated'.freeze
+  FILE_NOT_FOUND = 'file_not_found'.freeze
+  CHECKSUM_MISMATCH = 'checksum_mismatch'.freeze
 
   # @note Hash values cannot be modified without migrating any associated persisted data.
   # @see [enum docs] http://api.rubyonrails.org/classes/ActiveRecord/Enum.html
+  # TODO: Port over statuses to archive pres_copy model
   enum status: {
     OK_STATUS => 0,
     INVALID_MOAB_STATUS => 1,
     INVALID_CHECKSUM_STATUS => 2,
     ONLINE_MOAB_NOT_FOUND_STATUS => 3,
     UNEXPECTED_VERSION_ON_STORAGE_STATUS => 4,
     VALIDITY_UNKNOWN_STATUS => 6,
-    UNREPLICATED_STATUS => 7
+    UNREPLICATED_STATUS => 7,
+    FILE_NOT_FOUND => 8,
+    CHECKSUM_MISMATCH => 9
```

Comment:
how is CHECKSUM_MISMATCH diff from INVALID_CHECKSUM? Should we pick one to use in both types of PreservedCopies?